client/components/migration/feature-announcement.jsx

```diff
@@ -63,10 +63,24 @@ const FeatureAnnouncement = ({ translate, isEligable }) => {
 					status: 'inactive'
 				})
 			} );
+		const markMigrationStartedAPICall = () =>
+			fetch( getBaseURL() + 'wc/v1/connect/migration-flag', {
+				method: 'POST',
+				headers,
+				body: JSON.stringify({
+					migration_state: 2 // Check WC_Connect_API_Constants::MIGRATION_STATE_STARTED
```

Comment:
I thought about using a constant for `2` but also thought that we would have to create a mapping to map the PHP side, which seems a bit too complex for what we are trying to do here. I left a comment to indicate where the constant is coming from instead.